sos/plugins/logs.py

```diff
@@ -50,91 +41,41 @@ def setup(self):
-
-    def postproc(self):
-        self.do_path_regex_sub(
-            r"/etc/rsyslog*",
-            r"ActionLibdbiPassword (.*)",
-            r"ActionLibdbiPassword [********]"
-        )
-        self.do_path_regex_sub(
-            r"/etc/rsyslog*",
-            r"pwd=.*",
-            r"pwd=[******]"
-        )
-
-
-class RedHatLogs(Logs, RedHatPlugin):
-
-    option_list = [
-        ("log_days", "the number of days logs to collect", "", 3)
-    ]
-
-    def setup(self):
-        super(RedHatLogs, self).setup()
-        # NOTE: for historical reasons the 'messages' and 'secure' log
-        # paths for the RedHatLogs plugin do not obey the normal --all-logs
-        # convention. The rotated copies are collected unconditionally
-        # due to their frequent importance in diagnosing problems.
-        messages = "/var/log/messages"
-        secure = "/var/log/secure"
-
-        have_messages = os.path.exists(messages)
-
-        self.add_copy_spec(secure + "*")
-        self.add_copy_spec(messages + "*")
-
-        # collect three days worth of logs by default if the system is
-        # configured to use the journal and not /var/log/messages
-        if not have_messages and self.is_installed("systemd"):
-            try:
-                days = int(self.get_option("log_days"))
-            except ValueError:
-                days = 3
-            if self.get_option("all_logs") and self.get_options("since"):
-                since = self.get_options("since")
-            else:
-                since = "-%ddays" % days
-            self.add_journal(since=since)
-
-
-class DebianLogs(Logs, DebianPlugin, UbuntuPlugin):
-
-    option_list = [
-        ("log_days", "the number of days logs to collect", "", 7)
-    ]
-
-    def setup(self):
-        super(DebianLogs, self).setup()
-        days = int(self.get_option("log_days"))
-        journal = os.path.exists("/var/log/journal/")
         self.add_copy_spec([
+            "/etc/syslog.conf",
+            "/etc/rsyslog.conf",
+            "/etc/rsyslog.d",
+            "/var/log/boot.log",
+            "/var/log/installer",
+            "/var/log/unattended-upgrades",
+            "/var/log/messages*",
+            "/var/log/secure*",
             "/var/log/udev",
             "/var/log/dist-upgrade",
-            "/var/log/installer",
-            "/var/log/unattended-upgrades"
-            ])
+        ])
+
+        self.add_cmd_output("journalctl --disk-usage")
         self.add_cmd_output('ls -alRh /var/log/')
 
+        journal = os.path.exists("/var/log/journal/")
         if journal and self.is_installed("systemd"):
-            if self.get_option("all_logs"):
-                since = self.get_options("since")
-            else:
-                since = "-%ddays" % days
             self.add_journal(since=since)
-        else:
+            self.add_journal(boot="this", catalog=True)
+            self.add_journal(boot="last", catalog=True)
```

Comment:
Could you apply `since` option to both these calls? As discussed above, sosreport --since "-1 day" should not collect data from previous boot 3 days ago.

Reply:
I'd prefer that but @bmr-cymru wants an exception for those for RH historical reasons.  I think we've gone back and forth on the since there several times in this PR.  Let's just land it without the since and revisit it in a future PR after we see how the teams like the other changes?